### PR TITLE
V2.x window updates

### PR DESCRIPTION
**Pull request by hjelmn**

### CODE REVIEW

**Comment by hjelmn** [comment]
:bot:assign: @jsquyres

closes open-mpi/ompi#814

File: ompi/mpi/c/win_get_info.c [hunk 1]
```diff
@@ -27,6 +27,10 @@
 
 static const char FUNC_NAME[] = "MPI_Win_get_info";
 
+static void _win_info_set (ompi_info_t *info, const char *key, int set)
```
Comment (jsquyres) [inline ompi/mpi/c/win_get_info.c:30]: Symbols with "_" prefixes are supposed to be the domain of the compiler, not user functions.

Comment (hjelmn) [inline ompi/mpi/c/win_get_info.c:30]: Only in the _[A-Z] case. _[a-z] is meant to designate file scope.

Comment (hjelmn) [inline ompi/mpi/c/win_get_info.c:30]: See C99 § 7.1.3 bullets 1-2. The first designates `_[A-Z]*` and `__*` as reserved. The second explicitly allows `_[a-z]*` for file-scoped identifiers.

Comment (jsquyres) [inline ompi/mpi/c/win_get_info.c:30]: I stand corrected!

:+1:

**Comment by jsquyres** [comment]
:+1:

**Comment by jsquyres** [comment]
@hppritcha I think this is good to go

**Pull request merged**